### PR TITLE
instances of jquery found

### PR DESCRIPTION
**Pull request by HeroicHitesh**
**Instances**
Lines|Files
-----|-----
115 | curriculum/README.md
3 | curriculum/_templates/project.md
115 | curriculum/api/apis-and-json.md
13 | curriculum/data-visualization/data-visualization.md
69 | curriculum/express-js/server-side-project.md
13, 24 | curriculum/projects/eventonica/README.md
54 | curriculum/javascript/javascript-9-async.md
18, 53 | curriculum/javascript/restaurant recommender/RestaurantRecommender-part2.md
9 | curriculum/objectives/9.md
43 | curriculum/objectives/week-6-assessment.md
17, 21 | curriculum/projects/auth0-assessment.md
27, 30, 35 | curriculum/projects/eventonica/eventonica-part4-apis.md
59, 75 | curriculum/projects/eventonica/eventonica-part5-express-backend.md
5 | curriculum/projects/eventonica/eventonica-part6-postgres.md
13, 24 | curriculum/projects/eventonica/README.md
4, 5, 15, 88, 102 | curriculum/projects/eventonica/starter-code/index.html
28 | curriculum/projects/recipe-page/phase-1-html-prompt.md
29 | curriculum/projects/recipe-page/phase-2-css-prompt.md
30 | curriculum/projects/recipe-page/phase-3-bootstrap-prompt.md
20 | curriculum/projects/take-home-problems/frontend.md
17 | curriculum/react-js/react-part-2.md
30, 57 | curriculum/react-js/react.md
16 | curriculum/web/ajax.md
60 | curriculum/web/bootstrap.md
79 | curriculum/web/css.md
58 | curriculum/web/html.md

**Whole files** *(Files having more than 5 instances)*
- curriculum/web/jquery.md
- curriculum/javascript/first-js-oop-project.md
- curriculum/objectives/6.md
- curriculum/projects/eventonica/eventonica-part3-jquery-ui.md

**Folder**
- curriculum/web/jquery-programming-fundamentals/

### CODE REVIEW

**Comment by HeroicHitesh** [comment]
Resolves #1366

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Apprentices should memorize [**this**](Cheatsheet.md) within the first month.
 ### Week 4 - Intermediate JavaScript
 
 1. [HTML/JS Game Project](/projects/js-html-games.md)
-1. [jQuery](/web/jquery.md)
+1. [jQuery](/web/jquery.md)
```
Comment (Kundan28) [inline README.md:115]:
```suggestion
1. [jQuery](/web/jquery.md)
```
Please find some valid typo 😄

Comment (HeroicHitesh) [reply README.md:115]: I thought we need to delete this link so typo doesn't matter here😅
So, we need to find existing typos? I thought just create one and in the next PR of same kind remove that.😌

**Comment by HeroicHitesh** [comment]
@Kundan28 I guess pending-pr-revisions tag can be removed.
@alodahl please review and merge if found suitable at the earliest so that I can move on to the next issue.

**Comment by Kundan28** [comment]
> @Kundan28 I guess pending-pr-revisions tag can be removed.

Removed that tag.

**Comment by alodahl** [comment]
fixes  #1366

**Comment by Kundan28** [comment]
Updated the scoresheet too.

**Pull request merged**